context.copytostorage/addon.py

```diff
@@ -0,0 +1,229 @@
+import xbmc
+import xbmcgui
+import xbmcaddon
+import sys
+import os
+import errno
+import shutil
+import json
+
+reload(sys)
```

Comment:
Why do you need this @AbelTesfaye ?
You can simply define a magic comment at the top of the file: `# -*- coding: utf-8 -*-`

Reply:
I'm using it to copy unicode filenames. I tried using the magic comment but it didn't work.

Reply:
Could be improved but it can be done in a follow up update. I'll squash and merge if travis is happy.